pages/community/precice-workshop-2025.md

```diff
@@ -23,6 +23,27 @@ Keep watching this space for updates on registration, the workshop program, and
   The submission form will be available on March 1.
 - Registration deadline: July 15.
 
+## Registration
+
+The workshop includes two main parts:
+
+- Training course: September 8-9. Includes the [course](community-training.html) parts 1-5.
+- Main workshop: September 10-12. Includes a new part of the course (running preCICE on HPC systems) and [everything else](precice-workshop.html).
+
+Ticket prices:
+
+| Ticket                                         | Price   |
+| ---                                            | ---     |
+| Full ticket (incl. course)                     | 700 €   |
+| Full ticket (incl. course) - academic discount | 350 €   |
+| Main workshop (no course)                      | 500 €   |
+| Main workshop (no course) - academic discount  | 250 €   |
+| Holders of a [support license](community-support-precice.html) | 0 € (up to 3) |
+
+The academic discount concerns publicly-funded research institutes (any country).
```

Comment:
```suggestion
```
I would rather not specify details here. Many institutions incl industry-like players get public funding.

Reply:
FYI, that was a copy-paste of what we had in 2023, to avoid questions via email.